deployment/terraform/cdn.tf

```diff
@@ -152,7 +152,13 @@ resource "aws_cloudfront_distribution" "cdn" {
 
     lambda_function_association {
       event_type = "viewer-request"
-      lambda_arn = "${aws_lambda_function.redirect_to_s3_origin.qualified_arn}"
+      lambda_arn = aws_lambda_function.redirect_to_s3_origin.qualified_arn
+    }
+
+    lambda_function_association {
+      event_type   = "viewer-response"
+      lambda_arn   = aws_lambda_function.add_security_headers.qualified_arn
+      include_body = false
```

Comment:
These are the only changes that relate to my task.
The rest are just code formatting